.circleci/config.yml

```diff
@@ -274,6 +274,10 @@ commands:
               && sudo mv chromedriver /usr/local/bin/chromedriver \
               && sudo chmod +x /usr/local/bin/chromedriver \
               && chromedriver --version
+      - run:
+          name: Install Playwright dependencies
+          command: |
+            sudo apt-get update && sudo apt-get --yes install --no-install-recommends libgbm1
```

Comment:
This should be in the `e2e_test_playwright` command (they're separate commands for some reason for now, will be fixed in my future PR)